content/en/docs/installation/uninstallation.md

```diff
@@ -26,7 +26,7 @@ helm uninstall kyverno kyverno/kyverno -n kyverno
 
 ### Clean up Webhooks
 
-Kyverno by default will try to clean up all its webhooks when terminated. But in cases where its RBAC resources are removed first, it will lose the permission to do so properly.
+Kyverno by default will try to clean up all its webhooks when terminated. But in cases where its RBAC resources are removed first, it will lose the permission to do so properly. Kyverno also supports deletion using finalizers to ensure resource deletion in right order. Use `--autoDeleteWebhooks` in admission-controller and cleanup-controller deployment to enable this feature.
```

Comment:
>Kyverno by default will try to clean up all its webhooks when terminated.
Is this still true? The flag is set to false by default.

Reply:
Yeah it still tries to do it in code: https://github.com/kyverno/kyverno/blob/0a4f796af385449570beee8b518e3d27e53b4b35/pkg/webhooks/server.go#L243-L275

And we also delete hooks ourselves in pre-delete hook for helm